shoov/modules/custom/shoov_restful/plugins/restful/node/ShoovEntityBaseNode.php

```diff
@@ -53,6 +53,16 @@ protected function imageProcess($value) {
       }
       return $output;
     }
+
+    $defaultStyles = shoov_screenshot_image_default_styles();
```

Comment:
We have this in order not to hardcode the styles. If later we'll decide to add/change/remove some image styles (e.g. add one more size or change the name of the existing one) - we won't have to change it in different places - just download the features and add them to the field in restful class.